Examples/ObjectiveC/CacheManagementExample.m

```diff
@@ -0,0 +1,158 @@
+#import "CacheManagementExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleCacheManagement = @"CacheManagementExample";
+
+@interface CacheManagementExample () <MGLMapViewDelegate>
+
+@property (nonatomic) MGLMapView *mapView;
+@property (nonatomic) UIAlertController *alertController;
```

Comment:
Does this need to be a property? If it’s only used to check if something is already being presented, there may be a suitable property on the view controller already.